frontend/src/components/UserTables/LocationsTable.tsx

```diff
@@ -10,73 +10,84 @@ const Table = () => {
     const locationsData = await fetch('/locations')
       .then((res) => res.json())
       .then((data) => data.data);
-    setLocations(locationsData.map((location: any) => ({
+    const locs = locationsData.map((location: any) => ({
```

Comment:
Might be worth it just keeping this whole thing wrapped in `setLocations`. No need to save variables if you're just going to use them once.

Reply:
Yah, I originally had it like that, but I was running into some issues with my formatter so I set it like so. I'll move it back though.